File: Solution_Practice_I/python/prob1_3.py
while True:
    print(f"Ingrese el monto prestado, interes anual y el tiempo de años → (cantidad interes tiempo)")
    money, interest, time = map(float, input().split(" "))
    time = int(time)
    if(money>=0 and interest>=0 and time>=0):
        break
interest/=100
interest*=money
for i in range(0, time, 1):
    money+=interest
print(money)

File: Solution_Practice_I/java/prob1_3.java
import java.util.Scanner;
public class prob1_3 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        double money,interest;
        int time;
        do{
            System.out.println("Ingrese el monto prestado, interes anual y el tiempo de años");
            money=input.nextInt();
            interest=input.nextInt();
            time=input.nextInt();
        }while((money<0)||(interest<0)||(time<0));
        interest/=100;
        interest*=money;
        for (int i=1; i<=time;i++){
            money+=interest;
        }
        System.out.println(money);

        input.close();
    }
}
